worker_binder.go

```
@@ -0,0 +1,93 @@
+	}
+}
+
+func (qs *workerBinder[T, R]) handleQueueSubscription(action string, data []byte) {
+	switch action {
+	case "enqueued":
+		qs.worker.notifyToPullNextJobs()
+	}
+}
+
+func (qs *workerBinder[T, R]) BindQueue() Queue[T, R] {
+	defer qs.worker.start()
```

golangci-lint (1.64.8)

Error return value of `qs.worker.start` is not checked

(errcheck)